% Preamble
\documentclass[leqno]{article}[12pt]

% bibliography
\usepackage[style=numeric, sorting=none, backend=biber]{biblatex}
\addbibresource{sources.bib}

% Exclude the 'note' field from the bibliography
\AtEveryBibitem{\clearfield{note}}
\AtEveryBibitem{\clearfield{url}}
\setlength{\emergencystretch}{3em}
\usepackage{subcaption} % For subfigure environment

% Disable Hyphenation Globally
% \hyphenpenalty=10000
% \exhyphenpenalty=10000

\newcommand{\adjustedTotalPages}{\number\numexpr\pageref{LastPage}-1\relax}

% Packages
\usepackage{amsmath}
\usepackage{indentfirst}
\usepackage{ragged2e}
\usepackage{microtype}
\usepackage[hidelinks]{hyperref}
\usepackage{titlesec}
\usepackage{tocloft}
\usepackage{graphicx}
\usepackage{array}
\usepackage{float}
\usepackage{caption}
\usepackage{geometry}[margin=1.5cm]
\usepackage{setspace}
\usepackage{booktabs}
\usepackage{lastpage}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage{listings}
\usepackage{xcolor}
\usepackage{tabularx}
\usepackage{titling}

\lstset{
    language=Python,
    basicstyle=\ttfamily\small,
    keywordstyle=\color{blue},
    stringstyle=\color{red},
    commentstyle=\color{green!50!black},
    numbers=left,
    numberstyle=\tiny,
    stepnumber=1,
    numbersep=5pt,
    showstringspaces=false,
    tabsize=4,
    breaklines=true,
    frame=single,
    inputencoding=utf8,
    extendedchars=true,
    literate={},
    % Reduce spacing in code blocks
}

% Set the title format
\setlength{\droptitle}{-5em} % Adjust the vertical space above the title


% Better headers
\usepackage{fancyhdr}
\pagestyle{fancy}
\fancyhf{}
\fancyhead[L]{\nouppercase{\leftmark}}
\fancyfoot[L]{Raphaël Ribes} % Footer left
\fancyfoot[C]{}              % Footer center
\fancyfoot[R]{\thepage/18}      % Footer right

\renewcommand{\headrulewidth}{0.4pt} % Thin line under header
\renewcommand{\footrulewidth}{0.4pt} % Line above the footer
\geometry{
  bottom=2cm,
  footskip=1cm
}
% Make the footer smaller

% Set default paragraph indentation
\setlength{\parindent}{2em}

% Table of contents
\renewcommand{\contentsname}{Table of Contents}

% Change section numbering to Roman numerals
\renewcommand{\thesection}{\arabic{section}-}
\renewcommand{\thesubsection}{\Alph{subsection})}

% Document
\begin{document}
\onehalfspace

\begin{titlepage}
    \begin{center}
	    \includegraphics[height=3cm]{Page_de_garde/logo_UM}\hspace{0.2cm}
	    \includegraphics[height=3cm]{Page_de_garde/logo_fds_rond}\hspace{0.2cm}
	    \includegraphics[height=3cm]{Page_de_garde/Logo_Bioinfo}\hspace{0.2cm}

        {\Large Master Bioinformatique\\[1cm]}

        {\large HAU701I : Bibliographic Project\\[0.7cm]}

        \vspace{3cm}

        % Title with bars
        \hrulefill\\[0.4cm]
        {\fontsize{18}{22}\textbf{Evolution of Gene Network Analysis Methods: Towards an Approach Using Random Matrix Theory}}\\[0.4cm]
        \hrulefill
    \end{center}

    \vfill
    \noindent
    \begin{minipage}{0.4\textwidth}
      \begin{flushleft} \large
        \emph{Author :}\\
        Raphaël \textsc{Ribes}\\
      \end{flushleft}
    \end{minipage}
    \begin{minipage}{0.5\textwidth}
      \begin{flushright} \large
        \emph{Scientific supervision:} \\
        Dr. Jizhong \textsc{Zhou}\\
        \emph{Pedagogical tutor:} \\
        Dr. Konstantin \textsc{Todorov}\\
      \end{flushright}
    \end{minipage}
\end{titlepage}

% Define the page style for the first two pages
\fancypagestyle{firstpagestyle}{
    \fancyhf{} % Clear all headers and footers
    \renewcommand{\headrulewidth}{0pt} % Remove header line
    \renewcommand{\footrulewidth}{0pt} % Remove footer line
}

% Apply the custom style to the first page
\thispagestyle{firstpagestyle}

% Reset and start page numbering from the second page
\newpage

\tableofcontents
\thispagestyle{firstpagestyle}

\newpage

\section*{Acknowledgements}\label{sec:acknowledgements}
\noindent I would like to thank Dr. Jizhong Zhou for sharing his expertise and helping with the proofreading of this work.
I am also grateful to Dr. Konstantin Todorov for his guidance as my educational tutor.
Thank you both for your contributions to the completion of this work.
\thispagestyle{firstpagestyle}
\newpage
\setcounter{page}{1} % Start numbering from 1
\pagenumbering{arabic}
% Sections
\section{Introduction}\label{sec:introduction}
\input{introduction}
\vfill
\section{Network Approaches Applied In Genomic Biology}\label{sec:network-approaches-applied-in-genomic-biology}
\input{network-approaches-applied-in-genomic-biology}

\section{Random Matrix Theory}\label{sec:random-matrix-theory}
\input{random-matrix-theory}

\section{Molecular Ecological Network Analysis}\label{sec:molecular-ecological-network-analysis}
\input{molecular-ecological-network-analysis}

\section{Discussion}\label{sec:discussion}
\input{discussion}

\section{Conclusion}\label{sec:conclusion}
In conclusion, the integration of RMT into gene network analysis represents a pivotal step toward more robust and meaningful biological interpretations.
By addressing the limitations of traditional methods and embracing the complexity of biological systems, RMT-based approaches pave the way for a deeper understanding of gene interactions and their implications in molecular ecology.
Continued collaboration across disciplines will be essential to harness the full potential of these methodologies, driving innovation and discovery in the years to come.

\noindent The innovations that led to the creation of MENA should serve as great motivation.
A motivation that helps researchers continue to push the boundaries of what is possible in gene network analysis and for exploring the application of RMT to other domains of biology.
For example, when used in cellular biology, RMT provides a great tool for analyzing the inherent noise and sparsity in single-cell genomic datasets.
It has been applied on a model based on data of random noise, sparsity-induced artifacts, and genuine biological signals that RMT helped disentangle these components effectively.
Ultimately, RMT-based methods facilitate deeper insights into complex biological systems, and help researchers to enhance the reliability and the precision of their findings.


\newpage
\addcontentsline{toc}{section}{References}
\printbibliography

\newpage

\section*{Abstract}\label{sec:abstract}
Gene network analysis plays an important role in the understanding of biological systems, including development, disease, and ecology.
This study reviews the development of gene network analysis methods, comparing traditional approaches like equation-based and Bayesian methods with newer techniques based on Random Matrix Theory (RMT). Traditional methods often struggle with scalability, sensitivity to noise, and subjective thresholding.
In contrast, RMT offers a systematic and reliable way to determine thresholds automatically.
Its integration into the Molecular Ecological Network Analysis pipeline strengthens gene network construction by reducing noise, minimizing bias, and increasing robustness.
This approach improves the understanding of system modularity and dynamics, addressing the limitations of older methods.
This bibliography puts in light RMT's potential to transform gene network analysis and expand its applications in biology and ecology.


\vfill % Push content to the bottom

\section*{Keywords}
\noindent Gene Network, Random Matrix Theory, Molecular Ecology, Network Analysis
\thispagestyle{firstpagestyle}
\end{document}