Gene networks are among the most crucial tools to study in biological research today.
These methods have evolved from their first application in genomic biology.
Combined with ecological models, their performance got improved metrics like accuracy and interpretability over time.

In 1736, in the city of Königsberg known nowadays as Kaliningrad (\autoref{fig:seven-bridges-of-königsberg}), Leonhard Euler received a challenge from one of his friends.
The challenge, supposed to be a joke, was deceptively basic: could a person cross all seven bridges of the city exactly once without retracing their steps?
Euler took this joke very seriously, at the point were the solution is the origin of graph theory.
He approached this problem by abstracting the geography of Königsberg into a network of nodes and edges.
The landmasses were represented as nodes, and the bridges connecting them were represented as edges.
% insert a image
\begin{figure}[h!]
    \centering
    \includegraphics[width=0.75\textwidth]{konigsberg-1581-22} % Path to your image file
    \caption{Seven Bridges of Königsberg\cite{young_seven_2020}}
    \label{fig:seven-bridges-of-königsberg}
\end{figure}

\noindent Euler proved that the problem had no solution, laying down two key principles in the process:
\begin{enumerate}
    \item Nodes and Edges: Euler identified that the ability to traverse a network depends on the degree of each node (the number of edges connected to it).
    For a path that crosses each edge exactly once (an Eulerian path), all but two nodes must have an even degree.
    \item Graph Connectivity: The network must be connected, meaning all nodes must be reachable from any other node.
\end{enumerate}

\noindent In the case of Königsberg, all four nodes had an odd degree, making it impossible to traverse the network under the stated conditions.
This conclusion did not solve the Euler's friend sunday walk, but established the first theorem of graph theory.
\\\\
Jacob Moreno and Helen Jennings took this idea a step further in the 1930s, drawing social relationships with sociometric maps that would be some of the first systematic applications of network analysis to social science\cite{moreno_who_1934}.
Use of network analysis has been applied to other domains like physics\cite{kirchhoff_solution_1958} and chemistry\cite{arthur_mathematical_1896}, showing how versatile and hows impactful this field can be.
This is why, in this work, the evolution of gene network analysis will be discussed, with a particular focus on the use of Random Matrix Theory (RMT) to increase robustness during network construction.


Gene networks are intricate representations of interactions among genes and their products within biological systems\cite{oldham_conservation_2006}.
These networks, composed of nodes symbolizing genes and edges reflecting interactions\cite{barabasi_network_2004}, offer a system-wide perspective on cellular processes.
Researchers leverage these networks to investigate critical biological phenomena\cite{barabasi_network_2004}, such as development\cite{montoya_ecological_2006}, disease progression\cite{friedman_using_2000}, and evolutionary adaptations\cite{dunne_food-web_2002}.
Notably, gene networks are instrumental in identifying gene modules—highly connected clusters of genes that frequently correspond to functional units and hub genes, which play pivotal roles in maintaining cellular integrity\cite{zhang_general_2005}.
\\\\
Constructing a gene network can be approached in multiple ways, each with its strengths and limitations.
With differential equations in the equation-based models.
They look at gene interactions are inferred from differential equations that describe gene expression dynamics\cite{deng_molecular_2012}.
A more probabilistic approach, like Bayesian networks, uses models to estimate gene interactions based on prior knowledge and observed data\cite{gerstung_quantifying_2009}.
Finally, the use of correlation matrices derived from gene expression data in co-expression networks identify links between genes that exhibit strong statistical relationships\cite{zhang_general_2005}.
These methods often rely on determining appropriate thresholds to distinguish meaningful biological connections from random noise, a process that remains subjective and heavily influenced by prior knowledge or experiments.
Despite their utility, traditional network approaches face challenges such as scalability and their subjectivity, emphasizing the need for advanced methods to refine and automate the thresholding process\cite{deng_molecular_2012}.
\\\\
This bibliography takes you on a journey through the history of methods for analyzing gene networks, with particular focus on the game-changing application of RMT in providing greater robustness to network construction.
We will explore first, common network approaches in genomic biology, comparing their strengths and limitations.
Then we will approach the fundamentals of Random Matrix Theory, its applications in many-body systems with an example of application in quantum physics.
Finally, the integration of RMT into the Molecular Ecological Network Analysis pipeline will be discussed, showing its impact on network construction and the broader implications for biological research.