\subsection{Equation-Based Network Methods}\label{subsec:equation-based-network-methods}
Equation-based methods offer a structured approach to modeling the dynamics of gene regulatory networks using ordinary differential equations (ODEs) to describe mRNA concentrations over time.
Linearizing these ODEs around a steady-state point simplifies the analysis, enabling the representation of gene interactions through a connectivity matrix \(A\), where \(a_{ij}\) quantifies the influence of gene \(j\) on gene \(i\)\cite{deng_molecular_2012}.


\noindent To capture the system's responses to external changes, perturbations (\(b_i\)) are incorporated into the ODE framework, providing a means to simulate environmental or experimental variations\cite{yeung_reverse_2002}.
These perturbations allow researchers to explore the robustness and adaptability of the network.

\newpage
\noindent Various methods have been used to infer these networks:

\begin{itemize}

    \item \textbf{Singular Value Decomposition (SVD):} is a mathematical technique used to decompose a matrix into a product of three other matrices where
    \subitem $X_{M\times N}$ is the original data matrix with M experiments and N genes.
    \subitem $U$ and $V$ are orthogonal matrices, meaning that their transposes are equal to their inverses
    \subitem $U^T.U=V^T.V=I$ where $I$ is the identity matrix.
    \subitem W is a diagonal matrix containing singular values.

    It is used in the context of reverse-engineering gene networks to simplify complex data sets and extract essential information about gene interactions\cite{yeung_reverse_2002}.

    \item \textbf{Robust Regression:} is a method used to fit a hyperplane to a set of points that may contain outliers, with the goal of passing through as many points as possible.
    Combined with SVD, robust regression enhances the reconstruction of connectivity matrices by prioritizing sparsity and minimizing the impact of outliers\cite{yeung_reverse_2002}.

\end{itemize}


\noindent Despite their strengths, equation-based methods rely on assumptions such as the validity of the linear approximation, which may fail for large perturbations.
SVD is a powerful tool; however, this method provides a family of candidate networks that are consistent with the microarray data.
It does not choose one candidate as the best model.
Moreover, sparse network reconstruction demands careful experimental design to balance the number of perturbations with data quality\cite{yeung_reverse_2002}.

\noindent These approaches provide powerful tools for inferring gene regulatory networks by integrating theoretical models with experimental data, enabling iterative refinements and deeper insights into biological systems.

\vfill
\subsection{Bayesian Network Methods}\label{subsec:bayesian-network-methods}

Bayesian networks are probabilistic graphical models.
These networks help in exploring complex interdependencies such as gene expression patterns and the dynamics of cancer progression\cite{friedman_using_2000, gerstung_quantifying_2009}.
Graphs are used in Bayesian networks where edges have one direction, and there are no cycles: a directed acyclic graph or DAG.
These graphs are used to model joint probability distributions, where nodes represent variables and edges reflect conditional relationships.
Thanks to this method, bayesian networks enable a better understanding of processes like gene regulation and the accumulation of mutations\cite{friedman_using_2000}.

Making a network learn requires finding the network that most effectively represents the observed data.
To evaluate potential network structures, scoring functions are used like the Bayesian scores, for example.
Sparse Candidate algorithm restricts the search to a smaller subset of relevant candidate variables, thus addressing the computational challenges associated with heavy datasets.
The bootstrap method can also be used to estimate the statistical confidence in the features of already learned networks.
This can be done by generating perturbed versions of the original data, and this method enables rapid and resource-efficient algorithms\cite{friedman_using_2000}.
\newpage
These networks are great tools in certain domains of genomic biology, offering practical solutions in areas like gene expression analysis and cancer progression modeling
\begin{itemize}
    \item \textbf{Gene Expression Analysis:} they uncover gene interactions and transcriptional regulation mechanisms by analyzing statistical dependencies.
        By identifying Markov blankets, they determine variables that directly influence genes and suggest possible cause-and-effect relationships.
    \item \textbf{Cancer Progression modeling:} Specialized models such as Conjunctive Bayesian Networks (CBNs) and Hidden CBNs (H-CBNs) track the accumulation of genetic mutations and their dependencies, aiding in understanding cancer progression.
        H-CBNs further improve robustness by incorporating observation error models to account for technical noise.
\end{itemize}

\noindent However, bayesian networks rely on handling of priors and assumptions.
When working with small datasets, prior knowledge strongly influences the learning process.
While these networks can infer causal relationships under the Causal Markov Assumption, such interpretations should be made cautiously and require other types of validation.
For example, hybrid approaches that combine methods with clustering algorithms to learn models over "clustered" genes\cite{friedman_using_2000}.

\noindent In spite of these challenges, Bayesian networks provide robust statistical tools and computational efficiency for exploring complex genomic problems, such as gene regulation and disease progression.


\subsection{Relevance/Co-Expression Network Methods}\label{subsec:relevance-co-expression-network-methods}

The relevance/co-expression network method is an analytical method designed to determine functional relationships among genes by investigating their co-expression patterns across diverse conditions or sample sets.
First, the pairwise correlations between gene expression profiles is calculated, commonly using Pearson correlation coefficients, which serve as a measure of similarity\cite{zhang_general_2005}.
Indeed, these Pearson correlation coefficients quantify the strength and the direction of the linear relationship between two expression levels of two genes across different samples.
The Pearson correlation coefficient $r$ is calculated as:
\[r = \frac{\sum_{i=1}^{n} (x_i - \bar{x})(y_i - \bar{y})}{\sqrt{\sum_{i=1}^{n} (x_i - \bar{x})^2 \sum_{i=1}^{n} (y_i - \bar{y})^2}}\]
Where $x_i$, $y_i$ are the expression levels of two genes across samples and $\bar{x}$, $\bar{y}$ are the mean expression levels of the respective genes.
These correlations are then transformed into connection weights via an adjacency function, where soft-thresholding is preferred over hard-thresholding to retain biological nuances\cite{butte_discovering_2000, oldham_conservation_2006}.
Hard-Thresholding gives a one if connected and zero if not where soft-thresholding gives a continuous value between 0 and 1, corresponding to the correlation coefficient.
The resulting network comprises nodes, representing genes, and edges, reflecting the strength of their co-expression relationships.
By applying a suitable threshold, weaker connections are excluded, enabling the identification of gene clusters, or modules, with significant co-expression\cite{schmitt_elucidation_2004}.
\\\\
Such modules often highlight genes involved in shared biological pathways, revealing insights into regulatory mechanisms and system-level gene interactions\cite{horvath_analysis_2006}.
Relevance networks have been employed to link gene expression with phenotypic traits, such as drug susceptibility, giving researchers clues about gene roles in specific biological processes or responses to environmental stimuli\cite{butte_discovering_2000}.
From oncogenic signaling to evolutionary studies of co-expression networks across species, this approach has demonstrated its efficiency\cite{oldham_conservation_2006}.
\\\\
Among the various methods for constructing gene co-expression networks, the correlation-based relevance network method stands out for its simplicity and resilience to noise\cite{butte_discovering_2000}.
This method calculates pairwise correlations among genes and uses thresholds to filter out weak associations associated as noise, producing cleaner networks\cite{schmitt_elucidation_2004}.
However, in spite of its advantages, the reliance on arbitrary thresholds is a significant limitation.
Indeed, these thresholds are often chosen based on subjective judgment or convenience introducing bias and affect the reproducibility and objectivity of the resulting networks\cite{oldham_conservation_2006}.
Arbitrary thresholding not only impacts the detection of biologically relevant interactions but also questions the methodâ€™s capacity to reflect the true complexity of gene regulatory mechanisms\cite{gardner_reverse-engineering_2005}.
Addressing these limitations requires more systematic approaches to threshold selection.
These improvements would enhance the robustness and reliability of relevance network analyses.

\subsection{General Comparaison}\label{subsec:general-comparaison}

If we compare the three network methods, we can see that each has its strengths and weaknesses.
The robustness of Bayesian methods is due to their probabilistic nature.
Equation-based approaches on another hand are more sensitive to noise if they are done without a careful experimental design.
Bayesian and relevance methods are better for scalability and so for handling large datasets in a contrary to equation-based.
Relevance methods can be disappointing in terms of biological accuracy.
Their reliance on simplistic correlation metrics makes them potentially overlook nuanced interactions.
However, when it comes to ease of interpretation, relevance networks are the simplest to understand, followed by Bayesian networks.
Equation-based models, although powerful, are extremely challenging to interpret due to their mathematical intricacies.
\\\\
Because of its computational simplicity and the nature of microarray data (typically noisy, highly dimensional and significantly under-sampled)\cite{gardner_reverse-engineering_2005}, correlation-based relevance network method is most commonly used for identifying cellular networks.
It is important to address the limitations of arbitrary thresholding, so those network methods could provide a more comprehensive and biologically accurate representation of gene interactions.
This is exactly what MENA does, by integrating RMT to provide a more systematic and robust approach to threshold selection.